oc-includes/osclass/model/Search.php

```diff
@@ -1105,7 +1105,7 @@ public function doSearch($extended = true, $count = true)
             }
 
 	        if ( $extended ) {
-                return Item::newInstance()->extendData($items);
+                return Item::newInstance()->extendData($items,true);
```

Comment:
extendData method only accept one argument, I didn't see any change in that method.